### DISCUSSION TITLE
Please update tiny-curl

### DISCUSSION THREAD

**Discussion by OSPanel** [curl/curl#13456] [General]
### I did this

tiny-curl 8.4.0 was released on October 30, 2023.

### I expected the following

I was expecting to see tiny-curl of a similar version with the current curl 8.7.1.

### curl/libcurl version

curl 8.7.1

### operating system

Windows 10

**Comment by dfandrich** [comment]
Tiny curl is not updated at the same cadence as regular curl.

**Comment by bagder** [comment]
Is there anything in particular you want/need in a fresh tiny-curl release?

**Reply by OSPanel** [reply]
In one of the projects, Curl is used for API requests only over the http protocol (not https, smtp, mta, pop3, etc.) from the console BATCH application. Therefore, it would be nice to use a lightweight version - tinyurl.

And the update is necessary because Curl recently fixed the text output to the Windows console.

**Comment by dfandrich** [comment]
You can compile a fairly small regular curl yourself from source. If you really
need a new tinycurl version, you might be able to talk to WolfSSL about
arranging for a special release.
